ci/build.sh

```diff
@@ -4,5 +4,5 @@ set -euo pipefail
 
 GOCACHE="$PWD/go-build"
 
-go build -i -ldflags='-s -w' -o bin/build build/main.go
-go build -i -ldflags='-s -w' -o bin/detect detect/main.go
\ No newline at end of file
+GOOS=linux    GOARCH=amd64 go build -i -ldflags='-s -w' -o bin/build build/main.go
```

Comment:
This is really so that I can get quick usable things built locally inside my buildpack builder locally (and I assume adding those ENV vars don't hurt when you're already on that target arch)